corehq/apps/sofabed/migrations/0002_delete_models.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
any instincts as to whether this might be slow or something to be concerned with? they are huge tables...